bamboo/docker-compose.yml

```diff
@@ -49,6 +49,7 @@ services:
     image: $CUMULUS_BASE_IMAGE
     volumes:
       - /tmp/cumulus_unit_test_data:/tmp/cumulus_unit_test_data
+      - $HOME/.aws/credentials:/home/app/.aws/credentials
```

Comment:
What's the reason for this?     This may not be the right spot to add this mount - we probably  don't want to expose all possible AWS creds to the docker stack on <random dev machine> for security reasons.         We could consider adding it to the CI compose file if that's an environment specific adaptation given it's more controlled.

Reply:
that's a mistake leftover from me trying to explore was credentials.... it doesn't work either sorry